sdk/maps/Azure.Maps.Route/README.md

```diff
@@ -0,0 +1,266 @@
+# Azure Maps Route client library for .NET
+
+Azure Maps Route is a library that can find route to a location or point of interests.
+
+[Source code](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/maps/Azure.Maps.Route/src) | [API reference documentation](https://docs.microsoft.com/rest/api/maps/) | [REST API reference documentation](https://docs.microsoft.com/rest/api/maps/route) | [Product documentation](https://docs.microsoft.com/azure/azure-maps/)
+
+## Getting started
+
+### Install the package
+
+Install the client library for .NET with [NuGet](https://www.nuget.org/):
+
+```dotnetcli
+dotnet add package Azure.Maps.Route --prerelease
+```
+
+### Prerequisites
+
+> You must have an [Azure subscription](https://azure.microsoft.com/free/dotnet/) and [Azure Maps account](https://docs.microsoft.com/azure/azure-maps/quick-demo-map-app#create-an-azure-maps-account).
+
+To create a new Azure Maps account, you can use the Azure Portal, Azure PowerShell, or the Azure CLI. Here's an example using the Azure CLI:
+
+```powershell
+az maps account create --kind "Gen2" --disable-local-auth true --account-name "myMapAccountName" --resource-group "<resource group>" --sku "G2" --accept-tos
+```
+
+### Authenticate the client
+
+There are 2 ways to authenticate the client: Shared key authentication and Azure AD.
+
+#### Shared Key Authentication
+
+* Go to Azure Maps account > Authentication tab
+* Copy `Primary Key` or `Secondary Key` under **Shared Key Authentication** section
+
+```C# Snippet:InstantiateRouteClientViaSubscriptionKey
+// Create a MapsRouteClient that will authenticate through Subscription Key (Shared key)
+var credential = new AzureKeyCredential("<My Subscription Key>");
+MapsRouteClient client = new MapsRouteClient(credential);
+```
+
+#### Azure AD Authentication
+
+In order to interact with the Azure Maps service, you'll need to create an instance of the `MapsRouteClient` class. The [Azure Identity library](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/identity/Azure.Identity/README.md) makes it easy to add Azure Active Directory support for authenticating Azure SDK clients with their corresponding Azure services.
+
+To use AAD authentication, set `TENANT_ID`, `CLIENT_ID`, and `CLIENT_SECRET` to environment variable and call `DefaultAzureCredential()` method to get credential. `CLIENT_ID` and `CLIENT_SECRET` are the service principal ID and secret that can access Azure Maps account.
+
+We also need **Azure Maps Client ID** which can get from Azure Maps page > Authentication tab > "Client ID" in Azure Active Directory Authentication section.
```

Comment:
```suggestion
We also need an **Azure Maps Client ID** which can be found on the Azure Maps page > Authentication tab > "Client ID" in Azure Active Directory Authentication section.
```

We should consider linking to the Azure Maps page here.  The context of what that refers to is not clear.